src/components/EntityInfo.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div
+    v-if="!isAmalgamationSelectionRoute"
```

Comment:
Hiding entity info when if we're on the amalgamation selection panel

Reply:
I recommend moving your conditional to App.vue (which is where the other conditional code for this exists) -- keep it all in one place.

Reply:
Fixed, thanks.